### PR TITLE
Add Amplitude to analytics section.

### PR DESCRIPTION
**Pull request by paladin8**
See https://amplitude.com/startups

Thanks!

### CODE REVIEW

**Comment by dakshshah96** [comment]
Hey 👋 Thanks for the contribution! Please make the following change to comply with the [contribution guidelines](https://github.com/dakshshah96/awesome-startup-credits/blob/master/CONTRIBUTING.md):

> Additions should be added to the bottom of the relevant category.

The PR should be good to merge once that's done! 😄

**Comment by paladin8** [comment]
Oops, sorry! Thought it was alphabetical. Fixed now.

File: README.md [hunk 1]
```diff
@@ -28,6 +28,7 @@ A collection of awesome companies offering free/discounted plans for eligible st
 
 - [Mixpanel for Startups](https://mixpanel.com/startups/) - With access to Mixpanel’s most advanced features, startups can do complex behavioral analysis in seconds and  collaborate across teams to unlock faster growth.
 - [Segment Startup Program](https://segment.com/industry/startups/) - Eligible startups get full access to Segment's Team Plan (up to $25,000 value) free for up to two years.
+- [Amplitude Startup Scholarship](https://amplitude.com/startups) - Access to Enterprise features of Amplitude's behavioral platform for product analytics at 10% of the list price.
```
Comment (dakshshah96) [inline README.md]:
```suggestion
- [Amplitude Startup Scholarship](https://amplitude.com/startups) - The Scholarship includes access to Amplitude’s behavioral analytics platform for 12 months. You'll also receive Amplitude's full suite of Enterprise features, including behavioral cohorting and user path analysis for 90% off the list price.
```

**Comment by dakshshah96** [comment]
Thank you for your contribution 🎉

**Pull request merged**